pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -337,10 +348,22 @@ func addContent(ignitionConfig *igntypes.Config, content []byte, dst string, mod
 	return nil
 }
 
-func addFile(ignitionConfig *igntypes.Config, src string, dst string, mode *int) error {
+func addFile(ignitionConfig *igntypes.Config, src string, dst string, mode *int, data interface{}) error {
```

Comment:
this works, but `interface{}` leaves a bad taste. Instead of adding code to this simple function, we can treat `runtimes.conf` like another template, adding a new helper function much like `GetOCIHooksConfigContent`

Reply:
ok will create a separate method for it

Reply:
done